sources/packages/backend/env-setup.js

```diff
@@ -0,0 +1,17 @@
+import * as dotenv from "dotenv";
```

Comment:
This file is a refactor from the below code adding the recursive .env file.
![image](https://user-images.githubusercontent.com/61259237/219769885-94d4ee91-e648-4ffd-a9d1-b9c892bcf804.png)